web_searchbar_full_width/static/src/xml/web_searchbar_full_width.xml

```diff
@@ -1,15 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates>
-    <t t-extend="ViewManagerAction">
-        <t t-jquery="tr.oe_header_row_top td:nth-child(1)">
-            jQuery(this).attr('colspan', 4);
-        </t>
-        <t t-jquery="tr.oe_header_row_top td:nth-child(2)">
-            var $new_row = jQuery('<tr class="oe_header_row"/>');
-            jQuery(this)
-            .parents('tr')
-            .after($new_row);
-            $new_row.append(jQuery(this).attr('colspan', 4));
+    <t t-extend="ControlPanel">
+        <t t-jquery="div.oe-cp-search-view" t-operation="replace">
```

Comment:
@hbrunn  , As this is template inheritance... So, we have this [type of operations](https://www.odoo.com/documentation/9.0/reference/qweb.html#template-inheritance) , In that I've not found type that only adds class to element. So, I replace whole element.
And if I'm wrong then correct me...   Thanks... :)